pkg/aggregator/context_resolver.go

```diff
@@ -65,6 +65,15 @@ func (cr *ContextResolver) trackContext(metricSampleContext metrics.MetricSample
 	return contextKey
 }
 
+func (cr *ContextResolver) get(key ckey.ContextKey) (*Context, bool) {
```

Comment:
does this have any overhead? 🤔

Reply:
Doubtful, It's just a function call (which might be optimized).